.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -60,6 +60,16 @@ body:
     placeholder: "2.0.0-beta.0"
   validations:
     required: true
+
+- type: dropdown
+  attributes:
+    label: Public or Confidential Client?
+    description: Are you using PublicClientApplication or ConfidentialClientApplication? (Issues regarding msal-browser, msal-react and msal-angular are always "Public")
+    options: 
+      - "Public"
+      - "Confidential"
```

Comment:
Can there be a both option? Or n/a?

Reply:
We can do multi-select, which I added to the other templates. I felt that a bug should be more specific to one or the other but I can add multi-select here as well if we think that's valid

Reply:
Picking one would be helpful as if we allow "both" who would we assign it to for the initial review/investigation?

Reply:
Assign it to both teams and whoever sees it first can unassign as appropriate?

Reply:
I think multi-select (and having it be optional) is best, as there could be issues that affect both or none.

Reply:
Can you provide an example of an issue that would affect neither Public or Confidential? Not a fan of making this optional as it will result in many people not selecting anything even in cases where they should.

Reply:
Issue installing MSAL Node itself, for example.

Reply:
IMO that type of issue gets opened so rarely I'm willing to accept the tradeoff of being forced to pick one or both in that scenario in order to ensure all the other issues coming in are being filled out completely.

Reply:
Sure, fair enough.